packages/components/src/Content/Content.test.tsx

```diff
@@ -44,3 +44,17 @@ it("renders a Content with a small spacing", () => {
     </div>
   `);
 });
+
+it("renders with a semantic tag when a valid type is set", () => {
+  const { container } = render(<Content type="section">A section!</Content>);
+
+  expect(container).toMatchInlineSnapshot(`
+    <div>
+      <section
+        class="padded base"
+      >
+        A section!
+      </section>
+    </div>
+    `);
+});
```

Comment:
I know we still use snapshots on older test but i'd prefer if we don't add more of them (or help us clean them up). We don't like snapshots as much here.

Reply:
Ah good to know. Just going off what was there.

Reply:
I cleaned the rest up too.